app/src/main/java/org/p2p/wallet/receive/list/TokenListData.kt

```diff
@@ -0,0 +1,10 @@
+package org.p2p.wallet.receive.list
+
+import org.p2p.wallet.user.model.TokenData
+
+data class TokenListData(
```

Comment:
maybe you should add some context that this class is data for the screen
smth like `TokenListScreenDetails` `TokenListScreenData`
and im not sure that default parameters is a good choice here